### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by justdan0227** [TablePlus/TablePlus#3232]
Is there a way we could select 2 databases (TEST and PRODUCTION) and get a list of differences in the table structures between the two?

**Comment by huyphams** [comment]
Hi @justdan0227 we don't have that feature at the moment.

**Comment by akiratuti** [comment]
This feature would be very useful. I currently use DBeaver to do this. It would be great to not have to switch between apps.

**Comment by shwuhk** [comment]
I agree! Want this feature too

**Comment by hzn6426** [comment]
Yes, support it, now i use navicat to compare two database.  The Tableplus is the only missing and most useful function!!
